### PR TITLE
Min sdk downgrade to 14

### PR DESCRIPTION
**Pull request by langsmith**
This pr downgrades minSdk number from 15 to 14. 

This pr does this in the maps sdk overview docs, navigation docs, and plugins docs.

cc @cammace

### CODE REVIEW

File: src/pages/navigation/overview/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Before developing your app with the Navigation SDK, you'll need to add the SDK a
 
 1. Start Android Studio
 2. Open up your app's `build.gradle` file
-3. Make sure that your project's `minSdkVersion` is at API 15 or higher
+3. Make sure that your project's `minSdkVersion` is at API 14 or higher
```
Comment (Guardiola31337) [inline src/pages/navigation/overview/index.md]: This is not true yet 👀 https://github.com/mapbox/mapbox-navigation-android/blob/master/gradle/dependencies.gradle#L4

File: src/pages/plugins/overview/index.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Note that depending on the plugin you add, there might be required permissions a
 
 1. Start Android Studio
 2. Open up your application's `build.gradle`
-3. Make sure that your project's `minSdkVersion` is at API 15 or higher
+3. Make sure that your project's `minSdkVersion` is at API 14 or higher
```
Comment (Guardiola31337) [inline src/pages/plugins/overview/index.md]: This is not true yet https://github.com/mapbox/mapbox-plugins-android/blob/master/gradle/dependencies.gradle#L4

File: src/pages/navigation/overview/index.md (hunk 1 above)
Comment (langsmith) [reply src/pages/navigation/overview/index.md]: Ok, reverted to 15

File: src/pages/plugins/overview/index.md (hunk 2 above)
Comment (langsmith) [reply src/pages/plugins/overview/index.md]: Ok, reverted to 15

**Comment by langsmith** [comment]
Should be good to go @Guardiola31337

**Pull request merged**